src_c/line.c

```diff
@@ -230,6 +230,44 @@ pg_line_copy(pgLineObject *self, PyObject *_null)
                                  self->line.x2, self->line.y2);
 }
 
+static PyObject *
+pg_line_parallel_test(pgLineObject *self, PyObject *const *args, Py_ssize_t nargs)
```

Comment:
Maybe you forgot to change that, you could rename the C function to be in line with the python name:
```suggestion
pg_line_is_parallel(pgLineObject *self, PyObject *const *args, Py_ssize_t nargs)
```